### PR TITLE
fix: update IOT-dev-guide URLs in /public/help.

### PR DESCRIPTION
**Pull request by jstirnaman**
This PR repair a few links to documentation targets.

## Proposed Changes

_Briefly describe your proposed changes:_

## Checklist

- [x] Rebased/mergeable
- [x] Commit messages are in [semantic format](https://seesparkbox.com/foundry/semantic_commit_messages)
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: app/ui/public/help/DashboardPage.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 [_Source code for this page on GitHub_](https://github.com/bonitoo-io/iot-center-v2/blob/master/app/ui/src/pages/DashboardPage.tsx)
 
-For more, see the [*InfluxDB IoT dev guide*](https://influxdata.github.io/iot-dev-guide/pages/dashboard.html).
+For more, see the [_InfluxDB IoT dev guide_](https://influxdata.github.io/iot-dev-guide/dashboard.html).
```
Comment (sranka) [inline app/ui/public/help/DashboardPage.md:5]:
```suggestion
For more, see the [_InfluxDB IoT dev guide_](https://influxdata.github.io/iot-dev-guide/ui/dashboard.html).
```

**Review by sranka — CHANGES_REQUESTED** [review]
LGTM except for one URL. Thank you for improving the docs.

File: app/ui/public/help/DashboardPage.md (hunk 1 above)
Comment (jstirnaman) [reply app/ui/public/help/DashboardPage.md]: @sranka Fixed. Thanks - thought I tested them all.

**Comment by jstirnaman** [comment]
Hi, @sranka, could you review this again please?

**Review by sranka — APPROVED** [review]

**Pull request merged**